lib/components/search/search.tsx

```diff
@@ -130,6 +131,12 @@ const Search: React.FC<unknown> = () => {
         .container {
           visibility: hidden;
         }
+        .container:global(ul),
+        .container:global(li) {
+          padding: 0;
+          margin: 0;
+          list-style: none;
+        }
         .container:global(.input-container.search-input) {
```

Comment:
Here is not the latest code of master, rebase and then check the logic of the code here.

Reply:
https://react-7b3zuxmg2-geist-ui.vercel.app/en-us/components/input